modules/service/data-service.js

```diff
@@ -100,7 +100,18 @@ class DataService {
                 },
                 data: await this.workerPool.exec('JSONParse', [fileContent.toString()]),
             };
-            const nquads = await this.workerPool.exec('toNQuads', [assertion.data]);
+            if (method !== constants.SERVICE_API_ROUTES.PUBLISH || method !== constants.SERVICE_API_ROUTES.UPDATE) {
+                if (assertion.data['@id']) {
+                    assertion.metadata.UAL = `dkg://did.${this.config.blockchain[0].networkId}.${this.config.blockchain[0].hubContractAddress}/${assertion.data['@id']}`;
```

Comment:
Is this intentional change to use did.polygon.hubContract instead of did:polygon:hubContract?

Reply:
If i undrestood it correctly, it is because it is to look more like regular URL?